### PR TITLE
PORT-7728 | Add guide to the OpenCost integration showing how to bring cost information from AWS

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description
Added guide for CloudCost integration in OpenCost. Included guide to bring cost information for specific providers.

## Updated docs pages

Please also include the path for the updated docs

- OpenCost (`docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ import AdvancedConfig from '../../../generalTemplates/_ocean_advanced_configurat
 
 # OpenCost
 
-Our OpenCost integration allows you to import `cost` from your OpenCost instance into Port, according to your mapping and definition.
+Our OpenCost integration allows you to import `cost` and `cloudCost` from your OpenCost instance into Port, according to your mapping and definition.
 
 ## Common use cases
 
 - Map your monitored Kubernetes resources and cost allocations in OpenCost.
+- Map you cost allocations from different providers from OpenCost.
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:15]:
```suggestion
- Map you cost allocations from different cloud providers from OpenCost.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md [hunk 2]
```diff
@@ -597,6 +599,127 @@ resources:
...
 
+### CloudCost
+
+<details>
+<summary><b>Cost blueprint (click to expand)</b></summary>
+
+```json showLineNumbers
+{
+    "identifier": "openCostCloudcost",
+    "description": "This blueprint represents an instance of OpenCost CloudCost data",
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:610]:
```suggestion
    "description": "This blueprint represents cloud cost allocations from your OpenCost instance",
```
This blueprint represents cloud cost allocations from your OpenCost instance

**Review by PeyGis — CHANGES_REQUESTED** [review]
some minor comments

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md [hunk 3]
```diff
@@ -344,7 +345,8 @@ The integration configuration determines which resources will be queried from Op
 :::tip Supported resources
 The following resources can be used to map data from OpenCost, it is possible to reference any field that appears in the API responses linked below for the mapping configuration.
 
-- [`Cost`](https://www.opencost.io/docs/integrations/api-examples)
+- [`Cost`](https://www.opencost.io/docs/integrations/api-examples#allocation-examples)
+- [`CloudCost](https://www.opencost.io/docs/integrations/api-examples#cloudcost-examples)
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:349]:
```suggestion
- [`Cloud Cost`](https://www.opencost.io/docs/integrations/api-examples#cloudcost-examples)
```

**Review by PeyGis — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md [hunk 4]
```diff
@@ -7,11 +7,12 @@ import AdvancedConfig from '../../../generalTemplates/_ocean_advanced_configurat
 
 # OpenCost
 
-Our OpenCost integration allows you to import `cost` from your OpenCost instance into Port, according to your mapping and definition.
+Our OpenCost integration allows you to import `cost` and `cloudCost` from your OpenCost instance into Port, according to your mapping and definition.
 
 ## Common use cases
 
 - Map your monitored Kubernetes resources and cost allocations in OpenCost.
+- Map you cost allocations from different cloud providers from OpenCost.
```
Comment (talru1123) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:15]: Map your*

Comment (stavbernazport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:15]:
```suggestion
- Map your cost allocations from different cloud providers from OpenCost.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md [hunk 5]
```diff
@@ -384,7 +386,7 @@ The following resources can be used to map data from OpenCost, it is possible to
       port:
   ```
 
-  - **window** - Duration of time over which to query. Accepts: words like `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; Unix timestamps like `1578002645,1580681045`.
+  - **window** - Duration of time over which to query. Accepts: words like `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; Unix timestamps like `1578002645,1580681045`.If not specified, the default value is `"today"`
```
Comment (stavbernazport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:389]:
```suggestion
  - **window** - Duration of time over which to query. Accepts words like: `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; Unix timestamps like `1578002645,1580681045`. If not specified, the default value is `"today"`
```

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md [hunk 6]
```diff
@@ -597,6 +599,127 @@ resources:
...
+            amortizedCost: .amortizedCost.cost
+
+```
+
+</details>
+
+#### Selector Parameters
+The following parameters under the `selector` key can be used to manipulate which data is ingested:
+
+- **window** - Duration of time over which to query. Accepts: words like `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; Unix timestamps like `1578002645,1580681045`. If not specified, the default value is `"today"`
```
Comment (stavbernazport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md:691]:
```suggestion
- **window** - Duration of time over which to query. Accepts words like: `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; Unix timestamps like `1578002645,1580681045`. If not specified, the default value is `"today"`
```

**Review by stavbernazport — APPROVED** [review]
LGTM

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md (hunk 3 above)
Comment (stavbernazport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md]:
```suggestion
- [`Cloud Cost](https://www.opencost.io/docs/integrations/api-examples#cloudcost-examples)
```

**Pull request merged**